components/usage/pkg/apiv1/billing.go

```diff
@@ -92,6 +92,19 @@ func (s *BillingService) GetStripeCustomer(ctx context.Context, req *v1.GetStrip
 			}
 
 			logger.WithError(err).Error("Failed to lookup stripe customer from DB")
+
+			return nil, status.Errorf(codes.NotFound, "Failed to lookup stripe customer from DB: %s", err.Error())
+		} else if customer.InvalidBillingAddress == nil {
+			// Update field for old entries in db
+			stripeCustomer, err := s.stripeClient.GetCustomer(ctx, customer.StripeCustomerID)
```

Comment:
:right_anger_bubble: Would be really cool if we only had on place to put this... but also, if it's more effort, it's probably not worth it for Git pod now.

WDYT @jeanp413 ?